specification/cognitiveservices/data-plane/CustomTranslator/stable/v1.0/CustomTranslator.json

```diff
@@ -0,0 +1,4371 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Custom Translator",
+        "version": "1.0",
+        "description": "Documentation for the Microsoft Custom Translator API<br/>C# code sample: https://github.com/MicrosoftTranslator/CustomTranslatorApiSamples",
+        "contact": {
+            "name": "Microsoft Custom Translator",
+            "email": "custommt@microsoft.com"
+        }
+    },
+    "host": "cognitivegbldev.azure-api.net",
+    "basePath": "/translator/customtranslator",
+    "schemes": [
+        "https"
+    ],
+    "securityDefinitions": {
+        "apiKeyHeader": {
+            "type": "apiKey",
+            "name": "Ocp-Apim-Subscription-Key",
+            "in": "header"
+        },
+        "apiKeyQuery": {
+            "type": "apiKey",
+            "name": "subscription-key",
+            "in": "query"
+        }
+    },
+    "security": [
+        {
+            "apiKeyHeader": []
+        },
+        {
+            "apiKeyQuery": []
+        }
+    ],
+    "paths": {
+        "/api/texttranslator/v1.0/models/{id}/undeployhubmodel": {
+            "delete": {
+                "description": "Accepts a request to undeploy a Hub model deployed to API V3.",
```

Comment:
This appears not REST API.

Delete should apply on the `/models/{id)`

Reply:
api-version should not in path.

Also this is a stable without any prior preview?